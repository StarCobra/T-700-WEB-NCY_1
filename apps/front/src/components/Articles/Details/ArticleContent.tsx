import React from "react";
import "../../../style/article.scss";
import { Box } from "@mui/material";
import Content from "../../Content";

export default function ArticleContent() {
  return (
    <Box className="fullArticleContent">
      <Content content={"LOREM IPSUM"} />

      <Content title={"Regulatory Challenges:"} content={"LOREM IPSUM"} />

      <Content title={"Environmental Concerns:"} content={"LOREM IPSUM"} />

      <Content title={"Technological Challenges:"} content={"LOREM IPSUM"} />

      <Content title={"Market Volatility:"} content={"LOREM IPSUM"} />

      <Content title={"Conclusion:"} content={"LOREM IPSUM"} />
    </Box>
  );
}
