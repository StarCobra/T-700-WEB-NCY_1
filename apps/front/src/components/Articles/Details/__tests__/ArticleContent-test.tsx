import React from "react";
import { act, create } from "react-test-renderer";
import ArticleContent from "../ArticleContent";

describe("Components > Articles > Details > ArticleContent", () => {
  const tests = [
    {
      description: "generic",
      props: {},
    },
  ];
  tests.forEach((test) => {
    it(`should render the component ${test.description}`, async () => {
      let root;
      await act(async () => {
        root = create(<ArticleContent {...test.props} />);
      });
      expect(root).toMatchSnapshot();
    });
  });
});
