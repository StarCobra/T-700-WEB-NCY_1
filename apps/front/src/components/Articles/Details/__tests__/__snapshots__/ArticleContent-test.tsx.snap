// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Articles > Details > ArticleContent should render the component generic 1`] = `
<div
  className="fullArticleContent MuiBox-root css-0"
>
  <div
    className="contentContainer MuiBox-root css-0"
  >
    <p>
      LOREM IPSUM
    </p>
  </div>
  <div
    className="contentContainer MuiBox-root css-0"
  >
    <h3>
      Regulatory Challenges:
    </h3>
    <p>
      LOREM IPSUM
    </p>
  </div>
  <div
    className="contentContainer MuiBox-root css-0"
  >
    <h3>
      Environmental Concerns:
    </h3>
    <p>
      LOREM IPSUM
    </p>
  </div>
  <div
    className="contentContainer MuiBox-root css-0"
  >
    <h3>
      Technological Challenges:
    </h3>
    <p>
      LOREM IPSUM
    </p>
  </div>
  <div
    className="contentContainer MuiBox-root css-0"
  >
    <h3>
      Market Volatility:
    </h3>
    <p>
      LOREM IPSUM
    </p>
  </div>
  <div
    className="contentContainer MuiBox-root css-0"
  >
    <h3>
      Conclusion:
    </h3>
    <p>
      LOREM IPSUM
    </p>
  </div>
</div>
`;
